docs/use-cases/configure.md

```diff
@@ -3,6 +3,68 @@ title: "Build a simple smart machine"
 linkTitle: "Build simple smart machines"
 weight: 10
 type: "docs"
-layout: "empty"
-canonical: "build/configure/"
+description: "Build a simple robot in a few steps."
```

Comment:
Let's go for a bit more description since this is what search machines will also use for indexing
```suggestion
description: "Build a simple smart machine in a few steps using Viam's modular system of components and services without writing much or any code."
```